azure-iot-hub-devicesdk/azure/iot/hub/devicesdk/transport/abstract_transport.py

```diff
@@ -48,7 +48,7 @@ def disconnect(self, callback):
         pass
 
     @abc.abstractmethod
-    def enable_feature(self, feature_name, callback=None, qos=1):
```

Comment:
>, qos=1 [](start = 56, length = 7)

I agree with this (this should be decided at the feature level)  but I just want to make sure it's intentional given it seems unrelated.

Reply:
This was intentional, but for a slightly different reason -- QOS is an MQTT concept and this is an interface that is common for all transports.

---
In reply to: [267173737](https://github.com/Azure/azure-iot-sdk-python-preview/pull/64#discussion_r267173737) [](ancestors = 267173737)